app/assets/css/main.css

```diff
@@ -333,6 +338,15 @@ Navigation
   text-decoration: none;
 }
 
+.main .pagination em {
```

Comment:
On pagination section em tag found in the link belonging to the active page was italic. I thought it would be better to turn it normal text style in terms of its beign compatible with the passive link.

the section i changed is bellow
<img width="201" alt="pagination" src="https://cloud.githubusercontent.com/assets/3652404/14406194/3d2809ac-fea9-11e5-8fae-8f558b6748bf.png">